extensions/cornerstone/src/init.tsx

```diff
@@ -282,8 +291,11 @@ export default async function init({
 
   function elementEnabledHandler(evt) {
     const { element } = evt.detail;
+
     element.addEventListener(EVENTS.CAMERA_RESET, resetCrosshairs);
 
+    eventTarget.addEventListener(EVENTS.IMAGE_LOAD_FAILED, imageLoadFailedHandler);
+    eventTarget.addEventListener(EVENTS.IMAGE_LOAD_ERROR, imageLoadFailedHandler);
```

Comment:
what about remove listeneres?

Reply:
I moved these outside the enabled element handler. These are event target and they should be next to the other eventTarget addEventListener calls. This way we don't need to worry about removing the handlers.